NBA-Stats-Dashboard/src/index.css

```diff
@@ -41,6 +43,30 @@
 
 .navMenu {
   grid-area: navMenu;
+  padding: 20px 10px;
+  background-color: var(--blue);
+}
+
+.navItem {
+  margin: 20px;
+  list-style-type: none;
+  font-size: 1.2rem;
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  color: var(--white);
+  opacity: 0.9;
+}
+.navItem:hover {
+  opacity: 1;
+}
```

Comment:
This hover effect is quite subtle and easy to miss to go from opacity 0.9 to 1. We should consider things like changing the background color and text color of the navItem and such, just so that it is super obvious when each item is hovered over.

Reply:
Good idea! I've made changes to the CSS, but I think we can make the interface even better. Want to talk about the design?

Reply:
Yes, we will talk more about design (styling) later. For the next few PRs the functionality will be more important.